constellations/gemini-admin/src/semester/semesterEdit.js

```diff
@@ -147,16 +148,32 @@ export class SemesterEditPage extends React.Component {
                 {modalDiv}
                 <h2>{title}</h2>
 
-                <h4>Semester ID</h4>
-                <input
+                <InputText
+                    label="Semester ID"
+                    required={true}
```

Comment:
add a description of what valid semester_ids can be. They must be of the form `year_season` e.g. 2020_fall or 2020_winter